### PR TITLE
Document environment variables used to configure CORS and allowed_hosts

### PR DESCRIPTION
**Pull request by p-fruck**
Adds documentation for the environment variables introduced in https://github.com/inventree/InvenTree/pull/4355

Also, my editor removed trailing whitespaces in the config file. Are you fine that or should I amend the commit to remove the whitespace diff?

### CODE REVIEW

**Comment by matmair** [comment]
@p-fruck  the whitespaces are needed to force a newline in Markdown.

File: docs/start/config.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The InvenTree server tries to locate the `config.yaml` configuration file on sta
 !!! tip "Config File Location"
     When the InvenTree server boots, it will report the location where it expects to find the configuration file
 
-The configuration file *template* can be found on [GitHub](https://github.com/inventree/InvenTree/blob/master/InvenTree/config_template.yaml) 
+The configuration file *template* can be found on [GitHub](https://github.com/inventree/InvenTree/blob/master/InvenTree/config_template.yaml)
```
Comment (matmair) [inline docs/start/config.md:40]:
```suggestion
The configuration file *template* can be found on [GitHub](https://github.com/inventree/InvenTree/blob/master/InvenTree/config_template.yaml) 
```

File: docs/start/config.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ If not specified via environment variables, the fallback secret_key file (automa
 
 ## Database Options
 
-InvenTree provides support for multiple database backends - any backend supported natively by Django can be used. 
+InvenTree provides support for multiple database backends - any backend supported natively by Django can be used.
```
Comment (matmair) [inline docs/start/config.md:98]:
```suggestion
InvenTree provides support for multiple database backends - any backend supported natively by Django can be used. 
```

File: docs/start/config.md [hunk 3]
```diff
@@ -249,7 +249,7 @@ The logo and custom messages can be changed/set:
 | INVENTREE_CUSTOMIZE | customize.login_message | Custom message for login page | *Not set* |
 | INVENTREE_CUSTOMIZE | customize.navbar_message | Custom message for navbar | *Not set* |
 
-If you want to remove the InvenTree branding as far as possible from your end-user also check the [global server settings](../settings/global.md#server-settings). 
+If you want to remove the InvenTree branding as far as possible from your end-user also check the [global server settings](../settings/global.md#server-settings).
```
Comment (matmair) [inline docs/start/config.md:252]:
```suggestion
If you want to remove the InvenTree branding as far as possible from your end-user also check the [global server settings](../settings/global.md#server-settings). 
```

**Review by matmair — CHANGES_REQUESTED** [review]

**Comment by p-fruck** [comment]
> @p-fruck the whitespaces are needed to force a newline in Markdown.

Makes sense, I'll deactivate my formater next time. Applied your suggestions in 12830e4

**Review by matmair — APPROVED** [review]
looks good now @p-fruck !

**Comment by SchrodingersGat** [comment]
Cheers @p-fruck I'll merge this once the code PR is done.

**Comment by matmair** [comment]
Rendered pages look good

**Comment by SchrodingersGat** [comment]
Thanks @p-fruck

**Pull request merged**